src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -664,6 +664,13 @@
       "defaultValue": "true",
       "description": "Sets up Visual Studio Code Debugging for WebAssembly and Skia Desktop"
     },
+    "rider": {
```

Comment:
Let's make this new option turned on by default, we'll disable it by default in VS's wizard. Let's also add a readme file to explain what those files are.